README.rst

```diff
@@ -0,0 +1,112 @@
+Introduction
+============
+
+.. image:: https://readthedocs.org/projects/adafruit-circuitpython-bluefruitspi/badge/?version=latest
```

Comment:
Super nitpicky but the portion after the circuitpython piece should match the GitHub repo. So, I'd recommend dropping the friend from the github url. That way our adabot checks can match them all up.